Google.Api.Generator/Generation/ResourceNamesGenerator.cs

```diff
@@ -235,15 +237,14 @@ private IEnumerable<ClassDeclarationSyntax> ProtoMessagePartials()
                                 if (res.field.IsRepeated)
                                 {
                                     var repeatedTyp = Typ.Generic(typeof(ResourceNameList<>), one.ResourceNameTyp);
+                                    var s = Parameter(null, "s");
```

Comment:
Move this into the IsWildcard scope? I don't think it's needed outside it.

Reply:
`s` is also used in the `else` clause. The unhelpfully short name doesn't help.